src/main.cpp

```diff
@@ -99,6 +100,7 @@ int main(int argc, char *argv[])
         "-label %s",     &g_dataSetLabel, "Override the displayed dataset label for the next data file on the command line with the given name",
 
         "<SEPARATOR>", "\nInitial settings / remote commands:",
+        "-centeronlabel %s", &centerLabelName, "Center camera on the dataset with the given label name",
```

Comment:
The ordering here is what the user sees on the command line `-help`.  Let's put this down after to `-viewradius` since it's essentially setting the view position and radius.

Could be worth renaming it to start with `-view`, though nothing terribly convincing comes to mind... do you like any of `-viewatgeometry` `-viewtogeometry` `-viewtoward` `-viewlabel`?